### DISCUSSION TITLE
GrblHAL Web builder failing on probeing

### DISCUSSION THREAD

**Discussion by Klippert** [grblHAL/core#559] [Q&A] [answered]
I'm trying to build a new firmware for my Teensy with Phils breakout board (T41U5XBB) - and i would like 4 axis + my Y axis is with two motors. 
So far so good, though when i try to include probing, I get an error:

Command failed: pio run -etest -cjob.ini
src/networking/networking.c:87:9: warning: unknown option after '#pragma GCC diagnostic' kind [-Wpragmas]
#pragma GCC diagnostic ignored "-Wstringop-overflow"
^
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c:120:8: error: unknown type name 'on_probe_fixture_ptr'
static on_probe_fixture_ptr on_probe_fixture;
^
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c: In function 'probe_fixture':
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c:294:18: error: called object 'on_probe_fixture' is not a function or function pointer
status = on_probe_fixture(tool, at_g59_3, on);
^
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c:120:29: note: declared here
static on_probe_fixture_ptr on_probe_fixture;
^
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c:262:19: warning: unused variable 'probe' [-Wunused-variable]
probe_state_t probe = hal.probe.get_state();
^
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c: In function 'probe_protect_init':
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c:576:28: error: 'grbl_t {aka struct <anonymous>}' has no member named 'on_probe_fixture'
on_probe_fixture = grbl.on_probe_fixture;
^
.pio/libdeps/test/grblhal_probe_plugin/probe_plugin.c:577:9: error: 'grbl_t {aka struct <anonymous>}' has no member named 'on_probe_fixture'
grbl.on_probe_fixture = probe_fixture;
^
*** [.pio/build/test/lib896/grblhal_probe_plugin/probe_plugin.c.o] Error 1
========================= [FAILED] Took 36.82 seconds =========================

Any advice on how to make a build that works?

**Comment by terjeio** [comment, marked as answer]
Try again now, plugin code was out of sync with the core.

**Reply by Klippert** [reply]
Thanks, Building and download works like a charm.
